packages/core/src/lib/quote/quote.spec.ts

```
@@ -32,9 +32,9 @@ describe('quote', () => {
     expect(res.val).toBeTruthy();
   });
 
-  it('should get quote for strk-evm', async () => {
+  it.only('should get quote for strk-evm', async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)